planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -1369,7 +1374,7 @@ BehaviorModuleOutput GoalPlannerModule::planPullOverAsOutput(
     thread_safe_data_.clearPullOverPath();
 
     // update goal candidates
-    auto goal_candidates = thread_safe_data_.get_goal_candidates();
+    auto goal_candidates = goal_candidates_;
```

Comment:
there are no `goal_candidates_` variable?
`goal_planner_data_` has `goal_candidates`?

Reply:
I added `goal_candidates_` member to GoalPlannerModule